src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/index.tsx

```diff
@@ -118,36 +118,23 @@ export const DetailsModal: React.FC<ModalProps & DetailsModalProps> = ({
     setIsUnderwritingGuidelineHit,
   ] = React.useState(false)
 
-  const bundleEditQuote = async (
-    form: EditQuoteInput,
-    offerData: OfferData,
-  ) => {
-    const mainQuoteResult = await editQuote({
-      variables: {
-        input: form,
-      },
-    })
-    if (
-      hasEditQuoteErrors(mainQuoteResult) ||
-      isUnderwritingLimitsHit(mainQuoteResult)
-    ) {
-      return mainQuoteResult
-    }
-    const quoutesToUpdate = offerData.quotes.filter(
-      (quote) => quote.id !== form.id,
-    )
+  const editQuotes = async (form: EditQuoteInput, offerData: OfferData) => {
```

Comment:
🙌 Makes a whole lot of more sense! I don't understand why we did it like that in the first place? 🤔

Reply:
No it's a mystery 🤷‍♀️